jobs/ci-kubernetes-e2e-kubeadm-gce-1-7.env

```diff
@@ -0,0 +1,13 @@
+### job-env
+
+PROJECT=k8s-jkns-gci-gke-reboot-1-4
```

Comment:
are we intentionally reusing this project because we don't want to create a new one?